Dockerfile

```diff
@@ -1,120 +1,138 @@
 # Multi-stage Dockerfile for all dataset-viewer services and jobs
 # Build with: docker build --target <service_name> -t <tag> .
 
+ARG PYTHON_VERSION=3.12.11
+FROM python:${PYTHON_VERSION}-slim AS viewer
```

Comment:
Building the rust based libviewer as a wheel to not include the compiler toolchains in the final docker images.